File: net/ShopErp.App/Service/Restful/OperatorService.cs
using ShopErp.Domain;
using ShopErp.Domain.RestfulResponse;
using System;
using System.Collections.Generic;
using System.Linq;
using ShopErp.Domain.RestfulResponse.DomainResponse;

namespace ShopErp.App.Service.Restful
{
    public class OperatorService : ServiceBase<Operator>
    {
        public static Operator LoginOperator = null;

        public DataCollectionResponse<Operator> GetByAll()
        {
            return DoPost<DataCollectionResponse<Operator>>(null);
        }

        public void Login(string number, string password)
        {
            Dictionary<string, object> para = new Dictionary<string, object>();
            para["number"] = number;
            para["password"] = Utils.Md5Util.Md5(password);
            var ret = DoPost<LoginResponse>(para);
            OperatorService.LoginOperator = ret.op;
            ServiceContainer.AccessToken = ret.session;
        }

        public void Logout(string session)
        {
            Dictionary<string, object> para = new Dictionary<string, object>();
            para["session"] = session;
            DoPost<ResponseBase>(para);
        }

        public void ModifyPassword(long id, string password)
        {
            Dictionary<string, object> para = new Dictionary<string, object>();
            para["id"] = id;
            para["password"] = password;
            DoPost<ResponseBase>(para);
        }
    }
}

File: android/app/src/main/java/bjc/shoperp/service/restful/OperatorService.java
package bjc.shoperp.service.restful;

import java.util.HashMap;

import bjc.shoperp.domain.Operator;
import bjc.shoperp.domain.restfulresponse.domainresponse.LoginResponse;
import bjc.shoperp.utils.Md5Util;

/**
 * Created by hcq on 2018/2/15.
 */

public class OperatorService extends bjc.shoperp.service.restful.ServiceBase<Operator> {


    private static LoginResponse loginVirtual() {
        LoginResponse lr = new LoginResponse();
        lr.op = new Operator();
        lr.op.Number = "1001";

        lr.session = "abcdefg";
        return lr;
    }

    public Operator Login(String number, String password) throws Exception {
        HashMap<String, Object> para = new HashMap<>();
        para.put("number", number);
        para.put("password", Md5Util.Md5(password));
        LoginResponse lr = DoPost(LoginResponse.class, para, null);
        //LoginResponse lr = loginVirtual();
        ServiceContainer.AccessToken = lr.session;
        return lr.op;
    }

}
